test/parse_request.cc

```diff
@@ -104,6 +110,8 @@ constexpr float kDefaultBicycle_DrivewayPenalty = 300.0f;      // Seconds
 constexpr float kDefaultBicycle_AlleyPenalty = 60.0f;          // Seconds
 constexpr float kDefaultBicycle_GateCost = 30.0f;              // Seconds
 constexpr float kDefaultBicycle_GatePenalty = 600.0f;          // Seconds
+constexpr float kDefaultBicycle_PrivateAccessCost = 5.0f;      // Seconds
+constexpr float kDefaultBicycle_PrivateAccessPenalty = 750.0f; // Seconds
```

Comment:
@CuriOusQuOkka why is kDefaultBicycle_PrivateAccessPenalty 750 for bike but 450 for the other costing

Reply:
The values are temporary now, I'm working on tuning them. But in the beginning it was set so high, to have `kDefaultBicycle_GatePenalty < kDefaultBicycle_PrivateAccessPenalty`. As we assume that private gates should be penalized more than gates with no access information.

Reply:
I generally use a lower gate cost these days for bicycles so perhaps it can be lowered.

Reply:
Good point, I'll try it out.

Reply:
I've set default `gate_penalty` to 300 for bikes, tested routes didn't change. `private_access_penalty=450` looks good for bikes.